features/app_and_device_attributes.feature

```diff
@@ -0,0 +1,45 @@
+Feature: App and Device attributes present
+
+Scenario: App and Device info is as expected
+    When I run "AppAndDeviceAttributesScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the "Bugsnag-API-Key" header equals "12312312312312312312312312312312"
+    
+    # Device
+    
+    And the payload field "events.0.device.osName" equals "iOS"
+    And the payload field "events.0.device.jailbroken" is false
+    And the payload field "events.0.device.osVersion" matches the regex "\d+\.\d+"
+    And the payload field "events.0.device.manufacturer" equals "Apple"
+    And the payload field "events.0.device.locale" is not null
+    And the payload field "events.0.device.id" is not null
+    And the payload field "events.0.device.model" equals "iPhone10,4"
```

Comment:
This will only work for certain iPhone models and indeed the Buildkite iOS 11 test fails because it's "iPhone10,1".  Having searched existing features files for the model field, step `the payload field "device.model" matches the test device model` looks like what you want.